src/containers/Profile/Read/index.js

```diff
@@ -61,6 +69,11 @@ export class ReadProfile extends Component {
     const loading = isFetching || !success || updating;
     const followersResults = followers.profiles;
     const followingResults = following.profiles;
+    const results = articles.results ? articles.results : null;
+    console.log('====================================');
+    console.log(results);
+    console.log('====================================');
+    const userBookmarks = bookmarks.results ? bookmarks.results : null;
```

Comment:
You can do sth like `bookmarks.results && bookmarks.results` here

Reply:
Same up on `articles.results && articles.results`, try also doing some destructuring